rest/blip_api_test.go

```diff
@@ -4494,3 +4494,49 @@ func TestBlipLegacyAttachDocUpdate(t *testing.T) {
 		require.Equal(t, keyValueErr.DocumentID, v2Key)
 	}
 }
+
+func TestUnsubChanges(t *testing.T) {
+	base.SetUpTestLogging(t, base.LevelInfo, base.KeyAll)
+	rt := NewRestTester(t, &RestTesterConfig{guestEnabled: true})
+	defer rt.Close()
+
+	btc, err := NewBlipTesterClientOptsWithRT(t, rt, nil)
+	require.NoError(t, err)
+	defer btc.Close()
+
+	// Sub changes
+	err = btc.StartPull()
+	require.NoError(t, err)
+	resp := rt.updateDoc("doc1", "", `{"key":"val1"}`)
+	_, found := btc.WaitForRev("doc1", resp.Rev)
+	require.True(t, found)
+
+	activeReplStat := rt.GetDatabase().DbStats.CBLReplicationPull().NumPullReplActiveContinuous
+	require.EqualValues(t, 1, activeReplStat.Value())
+
+	// Unsub changes
+	unsubChangesRequest := blip.NewRequest()
+	unsubChangesRequest.SetProfile(db.MessageUnsubChanges)
+	err = btc.pullReplication.sendMsg(unsubChangesRequest)
+	require.NoError(t, err)
+	unsubChangesResponse := unsubChangesRequest.Response()
+	_, err = unsubChangesResponse.Body()
+	require.NoError(t, err)
```

Comment:
Doing this doesn't really match how the `BlipTesterClient` works - which is a higher level API that hides a lot of the BLIP/replication messaging.

You could just move this code into a method on `BlipTesterClient`, so you can call `require.NoError(t, btc.UnsubChanges())`, in the same way you call `StartPull()` to start them.